code/modules/supply/supply_pack.dm

```diff
@@ -32,3 +32,30 @@
 		//Add the name to the UI manifest
 		ui_manifest += "[initial(AM.name)]"
 	manifest += "</ul>"
+
+/datum/supply_packs/proc/create_package(atom/spawn_location)
```

Comment:
Can `spawn_location` ever be a non-turf atom?

Reply:
It shouldn't be, but AA suggested atom and I never got a response on if it should be a turf. Should I convert it to turf?

Reply:
It should be turf